panel/io/location.py

```diff
@@ -25,6 +25,46 @@
     from bokeh.server.contexts import BokehSessionContext
     from pyviz_comms import Comm
 
+def _get_location_params(protocol: str|None, host: str| None, uri: str| None)->dict:
```

Comment:
Completely unclear to me what you have changed here, and how this relates to reverse proxies.

Reply:
Sorry, see the original issue now.